cfme/tests/intelligence/reports/test_validate_chargeback_report.py

```diff
@@ -490,6 +491,7 @@ def test_validate_default_rate_cpu_usage_cost(chargeback_costs_default, chargeba
             break
 
 
+@pytest.mark.rhv1
```

Comment:
It's quite strange that this is T1, while other test cases in this module are T3. I generally agree with T3 for all the test cases. And if one of them is the most basic for this feature, let's move it to T2. But I cannot see this making it into T1.

Reply:
I can move it to T2. I set it to T1 as a representative test or the charge back functionality - which is an important feature AFAIU.

Reply:
It is true that I am not very familiar with this feature. It's probably important for customer, but it still does not feel like sanity of the product. If it's important feature, let's go with T2.

Reply:
ok, will change it to T2